templates/index.jinja2

```diff
@@ -0,0 +1,45 @@
+<!doctype html>
```

Comment:
same here.  Don't re-write all the html that makes up the whole page.  If you want to adjust your "header" tag, you have to do it in each and every template instead of inheriting it from a single place.  `DRY` is for templates too